labeller/labeller.go

```diff
@@ -506,3 +511,12 @@ func (l *Labeller) podInNamespaces(pod *corev1.Pod) bool {
 	}
 	return false
 }
+
+func (l *Labeller) handleNoop(obj, oldObj *secscanv1alpha1.ImageManifestVuln) {
+	if _, ok := obj.GetLabels()["imagemanifestvuln.example"]; ok && !reflect.DeepEqual(obj.Spec, oldObj.Spec) {
+		_, err := l.sclient.SecscanV1alpha1().ImageManifestVulns(obj.GetNamespace()).UpdateStatus(updateImageManifestVulnLastUpdate(obj))
```

Comment:
It's not really a noop if it's updating the status.

Reply:
Right, `noop` isn't the best wording. We're essentially trying to implement something that will pass the `operator-scorecard` test that assumes that a user will create an `ImageManifestVuln` from the `alm-examples` annotation on our CSV, and expects the `status` block to change if they update the `spec`. This Operator doesn't work like that, but we can add a special handler for it.